src/LocaleString.res

```diff
@@ -156,9 +162,12 @@ let localeStrings = [
     surchargeMsgAmount: str => `A surcharge amount of ${str} will be applied for this transaction`,
     surchargeMsgAmountForCard: str =>
       `A surcharge amount of upto ${str} will be applied for this transaction`,
+    surchargeMsgAmountForOneClickWallets: "Additional fee applicable",
```

Comment:
@prafulkoppalkar you had made a change wherein if the once click payment methods need additional dynamic fields (like state/ country), the payment method is shown besides card, and is no more one click. How is Surcharge handled for that?

Reply:
@ArushKapoorJuspay has handled that .. it will come below similar to other payment methods surcharge